datafusion/physical-plan/src/windows/mod.rs

```diff
@@ -90,8 +90,15 @@ pub fn create_window_expr(
             ))
         }
         WindowFunctionDefinition::AggregateUDF(fun) => {
-            let aggregate =
-                udaf::create_aggregate_expr(fun.as_ref(), args, input_schema, name)?;
+            // TODO: Ordering not supported for Window UDFs yet
```

Comment:
We should probably file a ticket for this as well -- I can do so as part of this PR's review